en/config/safety.md

```diff
@@ -321,9 +321,17 @@ This section lists the available emergency switches.
 
 ### Kill Switch
 
-A kill switch immediately stops all motor outputs (and if flying, the vehicle will start to fall)!
-The motors will restart if the switch is reverted within 5 seconds.
-After 5 seconds the vehicle will automatically disarm; you will need to arm it again in order to start the motors.
+A kill switch immediately stops all motor outputs — if flying, the vehicle will start to fall!
+
+[By default](#COM_KILL_DISARM) the motors will restart if the switch is reverted within 5 seconds, after which the vehicle will automatically disarm, and you will need to arm it again in order to start the motors.
+
+| Parameter                                                                                                | Description                                                                                |
+| -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| <a id="COM_KILL_DISARM"></a>[COM_KILL_DISARM](../advanced_config/parameter_reference.md#COM_KILL_DISARM) | Timeout value for disarming after kill switch is engaged. Default: `5` seconds (Disabled). |
```

Comment:
```suggestion
| <a id="COM_KILL_DISARM"></a>[COM_KILL_DISARM](../advanced_config/parameter_reference.md#COM_KILL_DISARM) | Timeout value for disarming after kill switch is engaged. Default: `5` seconds. |
```
I don't get it, what does "Disabled" mean here?

Reply:
Yes, copy paste error! Thanks.